rollup.config.js

```diff
@@ -19,19 +19,6 @@ const globals = {
 };
 // $FlowFixMe This should be inferred by Flow and manual casting does not work inside of this config.
 const external = Object.keys(globals);
-const babelOptions = () => {
-  let result = {
```

Comment:
Is this removal a necessary part of the rollup 1.x upgrade, or is this an  adjacent optimisation related to the [.babelrc](https://github.com/JedWatson/react-select/pull/3366/files#diff-e56633f72ecc521128b3db6586074d2c) changes? If it's the latter, then we should revert this

Reply:
You may keep here only `plugins: ['emotion']`. Other plugins will be merged from root config. External helpers are added out of the box now.

Reply:
Add `exclude: /node_modules/`

Reply:
@gwyneplaine revert this